ils_middleware/tasks/folio/login.py

```diff
@@ -1,19 +1,22 @@
 """FOLIO Operators and Functions for Institutional DAGs."""
 import json
+import requests
 
-from ils_middleware.tasks.folio.request import FolioRequest
 from airflow.providers.http.operators.http import SimpleHttpOperator
 
 
 def FolioLogin(**kwargs) -> SimpleHttpOperator:
     """Logs into FOLIO and returns Okapi token."""
-    username = kwargs.get("username")
-    password = kwargs.get("password")
+    username = kwargs["username"]
+    password = kwargs["password"]
 
-    return FolioRequest(
-        **kwargs,
-        task_id="folio_login",
-        endpoint="authn/login",
-        response_filter=lambda response: response.json().get("x-okapi-token"),
-        data=json.dumps({"username": username, "password": password}),
-    )
+    url = "https://okapi-folio.dev.sul.stanford.edu/authn/login"
```

Comment:
Could we pull this url from an Airflow Variable instead? (Maybe from the calling DAG?) This would give us the flexibility of supporting different Okapi endpoints.

Reply:
Oh whoops, meant to put that in there. Good catch.Will fix